cfme/storage/volume.py

```diff
@@ -402,10 +408,11 @@ def status(self):
             self.refresh()
         else:
             view.browser.refresh()
-
-        for item in view.entities.elements.read():
-            if self.name in item['Name']:
-                return str(item['Status'])
+        try:
+            ent = view.entities.get_entity(name=self.name, surf_pages=True)
+            return ent.data["status"]
```

Comment:
**Question** Will the key 'status' always be there, perhaps we should catch `KeyError` as well?

Reply:
Status should always be there as it's column in that table I am trying to access.